# [Use cv_bridge with ROS and Python3](https://stackoverflow.com/questions/49221565/unable-to-use-cv-bridge-with-ros-kinetic-and-python3)
` from cv_bridge.boost.cv_bridge_boost import getCvType
ImportError: dynamic module does not define module export function (PyInit_cv_bridge_boost) `

* `$ cd ~/catkin_workspace/install/lib/python3/dist-packages`

* ` $ sudo cp -r cv_bridge /usr/lib/python3.6/dist-packages/
  `

## 完成人脸识别　　下一步将人脸识别和追踪算法结合　
｀ 5.13日